nixos/tests/prosody.nix

```diff
@@ -0,0 +1,71 @@
+import ./make-test.nix {
+  name = "prosody";
+
+  machine = { config, pkgs, ... }: {
+    services.prosody.enable = true;
+    environment.systemPackages = let
+      sendMessage = pkgs.writeScriptBin "send-message" ''
+        #!{pkgs.python3.interpreter}
```

Comment:
`#!{pkgs.python3.interpreter}` -> `#!${pkgs.python3.interpreter}`?

Reply:
Damn, you caught me! I'm working on adding that simple test at this very moment and pushed that commit to test it on another machine / trying it with the borg build bot. I hoped to be finished before you'd take a look. :wink:

Reply:
Test is now finished! This was harder than expected, sorry for the inconvenience.